docs/api-docs/cart-and-checkout/open-source-checkout.md

```diff
@@ -67,6 +67,19 @@ You can upload a custom checkout to your store's server using WebDAV. The instru
 
 ## Installing custom checkouts  
```

Comment:
The placement of this callout seems a bit off with it at the top of the Installation section. Looking at the headers as steps, it seems like a better fit at the end of **Obtaining the JavaScript loader file** since you would do these steps before **Hosting a custom checkout**, though it makes the most sense under a header titled "Developing a custom checkout."